creator-node/src/dbManager.js

```diff
@@ -203,6 +203,48 @@ class DBManager {
       log(`completed in ${Date.now() - start}ms`)
     }
   }
+
+  /**
+   * Retrieves md5 hash of all File multihashes for user ordered by clock asc, optionally by clock range
+   *
+   * @param {Object} lookupKey lookup user by either cnodeUserUUID or walletPublicKey
+   * @param {Number?} clockMin if provided, consider only Files with clock >= clockMin (inclusive)
+   * @param {Number?} clockMax if provided, consider only Files with clock < clockMax (exclusive)
+   * @returns {Number} filesHash
+   */
+  static async fetchFilesHashFromDB({
+    lookupKey: { lookupCNodeUserUUID, lookupWallet },
+    clockMin = null,
+    clockMax = null
+  }) {
+    let subquery = 'select multihash from "Files"'
+
+    if (lookupWallet) {
```

Comment:
any reason to not make this just cnodeUserUUID here?

Reply:
see the call to this function in `processSync`
processSync currently handles all user state by wallet and would require an extra query to get cnodeUserUUID so its 1) faster to do it this way since its in a single query and 2) felt it was cleaner to abstract this away from processSync

Reply:
and yeah good point on check - will add

Reply:
done